components/SwipeListView.js

```diff
@@ -81,10 +81,10 @@ class SwipeListView extends Component {
 				{
 					...Component.props,
 					ref: row => this._rows[`${secId}${rowId}`] = row,
-					onRowOpen: _ => this.onRowOpen(secId, rowId, this._rows),
-					onRowDidOpen: _ => this.props.onRowDidOpen && this.props.onRowDidOpen(secId, rowId, this._rows),
+					onRowOpen: toValue => this.onRowOpen(secId, rowId, this._rows, toValue),
```

Comment:
Whatever changes you make to this could you also make to the `else` case. There are two ways to pass the `SwipeRow` and this will only affect one of them. Thanks!